lib/morse_translator.rb

```diff
@@ -1,43 +1,94 @@
-                    "d" => "-..",
-                    "e" => ".",
-                    "f" => "..-.",
-                    "g" => "--.",
-                    "h" => "....",
-                    "i" => "..",
-                    "j" => ".---",
-                    "k" => "-.-",
-                    "l" => ".-..",
-                    "m" => "--",
-                    "n" => "-.",
-                    "o" => "---",
-                    "p" => ".--.",
-                    "q" => "--.-",
-                    "r" => ".-.",
-                    "s" => "...",
-                    "t" => "-",
-                    "u" => "..-",
-                    "v" => "...-",
-                    "w" => ".--",
-                    "x" => "-..-",
-                    "y" => "-.--",
-                    "z" => "--..",
-                    "1" => ".----",
-                    "2" => "..---",
-                    "3" => "...--",
-                    "4" => "....-",
-                    "5" => ".....",
-                    "6" => "-....",
-                    "7" => "--...",
-                    "8" => "---..",
-                    "9" => "----.",
-                    "0" => "-----",
-                    " " => " "}
+    @dictionary = {
+        "A" => ".-",
+        "B" => "-...",
+        "C" => "-.-.",
+        "D" => "-..",
+        "E" => ".",
+        "F" => "..-.",
+        "G" => "--.",
+        "H" => "....",
+        "I" => "..",
+        "J" => ".---",
+        "K" => "-.-",
+        "L" => ".-..",
+        "M" => "--",
+        "N" => "-.",
+        "O" => "---",
+        "P" => ".--.",
+        "Q" => "--.-",
+        "R" => ".-.",
+        "S" => "...",
+        "T" => "-",
+        "U" => "..-",
+        "V" => "...-",
+        "W" => ".--",
+        "X" => "-..-",
+        "Y" => "-.--",
+        "Z"  => "--..",
+        "a" => ".-",
+        "b" => "-...",
+        "c" => "-.-.",
+        "d" => "-..",
+        "e" => ".",
+        "f" => "..-.",
+        "g" => "--.",
+        "h" => "....",
+        "i" => "..",
+        "j" => ".---",
+        "k" => "-.-",
+        "l" => ".-..",
+        "m" => "--",
+        "n" => "-.",
+        "o" => "---",
+        "p" => ".--.",
+        "q" => "--.-",
+        "r" => ".-.",
+        "s" => "...",
+        "t" => "-",
+        "u" => "..-",
+        "v" => "...-",
+        "w" => ".--",
+        "x" => "-..-",
+        "y" => "-.--",
+        "z" => "--..",
+        "1" => ".----",
+        "2" => "..---",
+        "3" => "...--",
+        "4" => "....-",
+        "5" => ".....",
+        "6" => "-....",
+        "7" => "--...",
+        "8" => "---..",
+        "9" => "----.",
+        "0" => "-----",
+        " " => " "}
+                    
+  end
+
+  def create_upper_case_keys
+    @dictionary.keys.map do |element|
+      element.upcase
+    end
+  end
+
+  def translate(input)
+    input_array = input.chars
+    input_array.map do |keys|
+      @dictionary.values_at(keys)
```

Comment:
This works, but there is an easier way to access a value from a specific key: `@dictionary[key]`